arelle/plugin/validate/EFM/config.xml

```diff
@@ -205,6 +203,45 @@
      standardTaxonomyDatabase="efmStandardTaxonomies.db"
      standardTaxonomyUrlPattern="http://www.fasb.org|http://xbrl.sec.gov|http://www.xbrl.org|http://taxonomies.xbrl.us|http://xbrl.us/us-gaap|http://xbrl.ici.org/rr|http://xbrl.ifrs.org/taxonomy"
      />
+  <DisclosureSystem
+     names="US SEC (Edgar Filing Manual, Liberal, All Years)|efm-nonblocking-all-years|efm-liberal-all-years"

```

Comment:
The first name (`US SEC (Edgar Filing Manual, Liberal, All Years)`) conflicts with the entry above `efm-liberal`